Assets/Scripts/UI/OptionsMenu.cs

```diff
@@ -3,14 +3,19 @@
 using UnityEngine;
 
 namespace OperationBlackwell.UI {
-	public class OptionsMenu : MonoBehaviour {
+	public class OptionsMenu : Menu {
 
-		public void Option1Clicked() {
-			Debug.Log("Option 1 clicked");
+		public void ToggleFullscreen() {
+			Screen.fullScreen = !Screen.fullScreen;
```

Comment:
Is the text on menu updating on this or has it become a on/off switch?

Reply:
No the text does nothing here now, this is just the actual logic for toggling fullscreen.

Reply:
This has been fixed in the meantime.